src/main/resources/config/application-localvc.yml

```diff
@@ -12,5 +12,5 @@ artemis:
         url: http://localhost:8000
         build-agent-git-username: buildjob_user # Replace with more secure credentials for production. Required for https access to localvc. This config must be set for build agents and localvc.
         build-agent-git-password: buildjob_password # Replace with more secure credentials for production. Required for https access to localvc. This config must be set for build agents and localvc. You can also use an ssh key
-
+        use-version-control-access-token: true
```

Comment:
Is it good to define default values in artemis and localvc ? maybe move the artemis config to gitlab as well.